tests/test_apidoc.py

```diff
@@ -118,6 +120,15 @@ def test_pep_0420_disabled_top_level_verify(make_app, apidoc):
     print(app._status.getvalue())
     print(app._warning.getvalue())
 
+@pytest.mark.apidoc(coderoot=(rootdir / 'root' / 'trailing_underscore'))
```

Comment:
Yes, test for `pep_0420` should be also separated.
Historically, we had added some mark-up examples to `tests/root`. As a result, testing time had became very slow. Because it was built as 20+ times in our testcase.

>Also, isn't this a case that everything should be testing?

Should we test this with all builders? Is this case needed to be built?
Sphinx has already many builders and extensions. so we need to add new testdata to `tests/root` carefully to prevent combination-explosion.